bin/helpers/utils.js

```diff
@@ -1012,3 +1012,13 @@ async function processExitHandler(exitData){
   await this.stopLocalBinary(exitData.bsConfig, exitData.bsLocalInstance, exitData.args);
   process.exit(0);
 }
+
+exports.fetchZipSize = (fileName) => {
+  try {
+    let stats = fs.statSync(fileName)
+    return stats.size; // in bytes
+  }
+  catch(err) {
+    return "file not present"
```

Comment:
I'd suggest to keep this 0 to have consistency of data type (for ex, someone running a P90 or other mathematical function on size of zip on BQ would expect size to be always a number and not string).